test-files/cmd/glacial/DiffConflatePartialLinearMatchTest.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
```

Comment:
This is the test corresponding to the output image examples I'm about to post in the PR.